index.bs

```diff
@@ -632,19 +632,281 @@ Issue(12): Propose control protocol for Remote
+
+These parties may attempt to learn cryptographic parameters by observing data
+flows on the QUIC connection, or by observing cryptographic timing.
+
+### Active Network Attackers ### {#active-network-attackers}
+
+In addition, all parties connected to the LAN will be able to manipulate data
+exchanged between controllers and receivers.  They will be able to inject
+traffic into existing QUIC connections and attempt to inititate new QUIC
+connections.  These abilities can be used to attempt the following:
+
+*   Impersonate a new receiver or one already trusted by the user, in an attempt
+    to convince the user to authenticate to it.
+*   Connect to a receiver and query its capabilities.
+*   Connect to and control a presentation or remote playback, or extract data
+    from the application state of the presentation or remote playback.
+    
+One particular attack of concern is misconfigured or compromised routers that
+expose local network devices (such as Open Screen Protocol controllers and
+receivers) to the Internet.  This vector of attack has been used by malicious
+parties to take control of printers and smart TVs by connecting to local network
+services that would normally be inaccessible from the Internet.
+
+### Denial of Service ### {#denial-of-service}
+
+Parties with connected to the LAN may attempt to deny access to Open Screen
+Protocol controllers and receivers.  For example, an attacker my attempt to open
+a large number of QUIC connections to a receiver in an attempt to block
+legitimate connections or exhaust the receiver's system resources.  They may
+also multicast spurious DNS-SD records in an attempt to exhaust the cache
+capacity for mDNS listeners, or to get listeners to open a large number of bogus
+QUIC connections.
+
+### Same-Origin Policy Violations ### {#same-origin-policy-violations}
+
+The Presentation API allows cross-origin communication between controlling pages
+and presentations with the consent of each origin (through their use of the
+API).  This is similar to cross-origin communication via {{Window/postMessage()}}
+with a target origin of `*`.  However, the Presentation API does not convey
+source origin information with each message.  Therefore, the Open Screen
+Protocol does not convey origin information between controllers and receivers.
+
+The [=presentation ID=] carries some protection against unrestricted
+cross-origin access; but, rigorous authentication of the parties connected by a
+{{PresentationConnection}} must be done at the application level.
+
+Open Screen Protocol Considerations {#security-privacy-questions}
 -----------------------------------
 
-Threat model {#security-threat}
+### Personally Identifiable Information & High-Value Data ### {#personally-identifiable-information}
+
+The following data exchanged by the protocol can be personally identifiable
+and/or high value data:
+
+1. Presentation URLs and availability results
+1. Presentation IDs
+1. Presentation connection IDs
```

Comment:
Why are connection IDs valuable?

Reply:
A connection id maps onto a PresentationConnection (i.e. a player/controller in a multiplayer game).  If you expose them, one connection could impersonate another.